packages/grafana-ui/src/components/uPlot/Plot.tsx

```diff
@@ -72,13 +69,19 @@ export const UPlotChart: React.FC<PlotProps> = props => {
 
   // Memoize plot context
   const plotCtx = useMemo(() => {
-    return buildPlotContext(Boolean(plotInstance.current), canvasRef, props.data, registerPlugin, getPlotInstance);
+    return buildPlotContext(canvasRef, props.data, registerPlugin, getPlotInstance);
   }, [plotInstance, canvasRef, props.data, registerPlugin, getPlotInstance]);
 
   return (
     <PlotContext.Provider value={plotCtx}>
-      <div ref={plotCtx.canvasRef} data-testid="uplot-main-div" />
-      {props.children}
+      <div
+        className={css`
+          position: relative;
```

Comment:
really dislike these inline emotion stiles (I do them as well sometimes), which hurts the readability of the jxs markup, wonder if should add a utility emotion classNames for some common flex & css props.